recipes-asteroid/qml-asteroid/qml-asteroid-native_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "QML components for AsteroidOS"
```

Comment:
This recipe would be better named asteroid-generate-desktop-native. The recipe as-is gives the impression that it builds qml-asteroid but it actually does not